libs/api/domains/cms/src/lib/cms.contentful.service.ts

```diff
@@ -286,8 +286,9 @@ export class CmsContentfulService {
       year !== undefined && month !== undefined
         ? new Date(year, month, 1)
         : new Date()
-    const toDate = new Date(fromDate.getTime())
+    fromDate.setDate(fromDate.getDate() - 1)
 
+    const toDate = new Date(fromDate.getTime())
```

Comment:
It's not obvious to me what we are doing here, mind explaining why this is needed?
And maybe add it as a comment?

Reply:
The "upcoming auctions" used to show everything from the current datetime, so an auction at 12:00 would disappear at 12:01.

This makes the from date one day before.

I can add a comment in a fix PR I am working on for `main`. But this is the release branch so it would not be very useful :p